### ISSUE TITLE
"View Specifications" button on home screen not working

### ISSUE THREAD

**Issue by jbaird123** [t3-innovation-network/desm#185]
Clicking "View Specifications" on home screen results in blank screen with flashing image.

![image](https://user-images.githubusercontent.com/2940423/168130941-df6a2d3a-c08f-48e7-b35c-7c0b8a0466b1.png)

**Comment by excelsior** [comment]
@jbaird123 The problem was, your user didn't belong to any organization. I assigned you to the only organization we have on staging, Credential Registry.

**Comment by jbaird123** [comment]
@excelsior - This user should not belong to any organization.  It was supposed to be an administrative user.

**Comment by excelsior** [comment]
Yup, there was a bug: the role was called `administrator`, but in the code the `super admin` name was sought for.
